server/src/test/java/org/elasticsearch/cluster/coordination/PublicationTransportHandlerTests.java

```diff
@@ -232,7 +233,10 @@ public RecyclerBytesStreamOutput newNetworkBytesStream() {
                     VersionUtils.randomCompatibleVersion(random(), Version.CURRENT)
                 );
                 allNodes.add(node);
-                nodeTransports.put(node, node.getVersion().transportVersion);
+                nodeTransports.put(
+                    node,
+                    TransportVersionUtils.randomVersionBetween(random(), TransportVersion.MINIMUM_COMPATIBLE, TransportVersion.CURRENT)
```

Comment:
I don't think it matters here that Version and TransportVersion are not synced up, this is only used for a single serialisation stream